control-plane/agents/src/bin/core/controller/reconciler/node/nexus.rs

```diff
@@ -72,20 +75,73 @@ async fn republish_volume(
     Ok(())
 }
 
+async fn find_shutdown_volumes(context: &PollContext, node_id: &NodeId) -> Result<(), SvcError> {
+    let draining_starttime = context.specs().get_node_draining_timestamp(node_id).await?;
+
+    if draining_starttime.is_some() {
+        let elapsed = draining_starttime.unwrap().elapsed();
```

Comment:
```suggestion
    if let Some(draining_starttime) = draining_starttime else {
        let elapsed = draining_starttime.elapsed().unwrap_or(DRAINING_VOLUME_TIMOUT_SECONDS);
```
or you can short-circuit
```suggestion
    let Some(draining_starttime) = draining_starttime else {
      return Ok(());
    }
```

Reply:
fixed using the short-circuit